uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -245,7 +245,8 @@ private fun FeedScreen(
                         onDragStopped = onDragStopped,
                         draggableState = draggableState,
                         robotAnimValue = robotAnimValue,
-                        isListFinished = isListFinished
+                        isListFinished = isListFinished,
+                        scaffoldState = scaffoldState
```

Comment:
Can we pass `scaffoldState.isRevealed` instead of scaffoldState for Don't let FeedList know about the bottom sheet? 🐑